src/main/java/math/FibonacciSnippet.java

```diff
@@ -42,4 +45,20 @@ public static int fibonacci(int n) {
       return fibonacci(n - 1) + fibonacci(n - 2);
     }
   }
+
+  /**
+   * Example of what an iterative implementation of Fibonacci looks like.
+   *
+   * @param number given number
+   * @return fibonacci number for given n
+   */
+  public static int interactiveFibonacci(int number) {
```

Comment:
Shouldn't the method be called `iterativeFibonacci`?

Reply:
I ended up not realizing the error, adjusted